pyOpenMS nightly

It seems the nightly tests fail without an error: http://cdash.openms.de/buildSummary.php?buildid=166990

But to the actual issue:
When I try to run
`make pyopenms`
in the very same configured build folder from where the results are uploaded to CDash on our Mac, I run into the following error (first CMake lines omitted):
```
INFO:root:parse /Users/aiche/NIGHTLY/HEAD/src/pyOpenMS/pxds/ConsensusMapNormalizerAlgorithmMedian.pxd
Traceback (most recent call last):
  File "create_cpp_extension.py", line 44, in <module>
    decls, instance_map = autowrap.parse(pxd_files, ".")
  File "/usr/local/lib/python2.7/site-packages/autowrap/__init__.py", line 51, in parse
    return DeclResolver.resolve_decls_from_files(files, root)
  File "/usr/local/lib/python2.7/site-packages/autowrap/DeclResolver.py", line 172, in resolve_decls_from_files
    decls.extend(PXDParser.parse_pxd_file(full_path))
  File "/usr/local/lib/python2.7/site-packages/autowrap/PXDParser.py", line 462, in parse_pxd_file
    result.append(handler(body, lines, path))
  File "/usr/local/lib/python2.7/site-packages/autowrap/PXDParser.py", line 264, in parseTree
    decl = MethodOrAttributeDecl.parseTree(att, lines, pxd_path)
  File "/usr/local/lib/python2.7/site-packages/autowrap/PXDParser.py", line 377, in parseTree
    tt = _extract_type(arg.base_type, argdecl)
  File "/usr/local/lib/python2.7/site-packages/autowrap/PXDParser.py", line 188, in _extract_type
    return CppType(base_type.name, template_parameters, is_ptr, is_ref, is_unsigned, is_long, is_const=is_const)
AttributeError: 'CConstTypeNode' object has no attribute 'name'
make[3]: *** [src/pyOpenMS/CMakeFiles/pyopenms_create_cpp] Error 1
make[2]: *** [src/pyOpenMS/CMakeFiles/pyopenms_create_cpp.dir/all] Error 2
make[1]: *** [src/pyOpenMS/CMakeFiles/pyopenms.dir/rule] Error 2
make: *** [pyopenms] Error 2
```

--- a/src/pyOpenMS/pxds/ConsensusMapNormalizerAlgorithmMedian.pxd
+++ b/src/pyOpenMS/pxds/ConsensusMapNormalizerAlgorithmMedian.pxd
@@ -1,15 +1,6 @@
 from libcpp.vector cimport vector as libcpp_vector
 from ConsensusMap cimport *
-
-cdef extern from "<OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMedian.h>" namespace "OpenMS":
-
-    cdef cppclass ConsensusMapNormalizerAlgorithmMedian:
-
-        ConsensusMapNormalizerAlgorithmMedian() nogil except +
-        ConsensusMapNormalizerAlgorithmMedian(ConsensusMapNormalizerAlgorithmMedian) nogil except + #wrap-ignore
-
-        Size computeMedians(ConsensusMap & input_map, libcpp_vector[double] & medians, const String & acc_filter, const String & desc_filter) nogil except +
-        void normalizeMaps(ConsensusMap & input_map, NormalizationMethod method, const String & acc_filter, const String & desc_filter) nogil except +
+from ConsensusMapNormalizerAlgorithmThreshold cimport *
 
 cdef extern from "<OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMedian.h>" namespace "OpenMS::ConsensusMapNormalizerAlgorithmMedian":
 
@@ -17,3 +8,11 @@ cdef extern from "<OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMe
         NM_SCALE,
         NM_SHIFT
 
+cdef extern from "<OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmMedian.h>" namespace "OpenMS":
+    cdef cppclass ConsensusMapNormalizerAlgorithmMedian:
+
+        ConsensusMapNormalizerAlgorithmMedian() nogil except +
+        ConsensusMapNormalizerAlgorithmMedian(ConsensusMapNormalizerAlgorithmMedian) nogil except + #wrap-ignore
+
+        Size computeMedians(ConsensusMap & input_map, libcpp_vector[double] & medians, String & acc_filter, String & desc_filter) nogil except +
+        void normalizeMaps(ConsensusMap & input_map, NormalizationMethod method, String & acc_filter, String & desc_filter) nogil except +

--- a/src/pyOpenMS/pxds/ConsensusMapNormalizerAlgorithmThreshold.pxd
+++ b/src/pyOpenMS/pxds/ConsensusMapNormalizerAlgorithmThreshold.pxd
@@ -8,6 +8,6 @@ cdef extern from "<OpenMS/ANALYSIS/MAPMATCHING/ConsensusMapNormalizerAlgorithmTh
         ConsensusMapNormalizerAlgorithmThreshold() nogil except +
         ConsensusMapNormalizerAlgorithmThreshold(ConsensusMapNormalizerAlgorithmThreshold) nogil except + #wrap-ignore
 
-        libcpp_vector[double] computeCorrelation(ConsensusMap & input_map, double ratio_threshold, const String & acc_filter, const String & desc_filter) nogil except +
+        libcpp_vector[double] computeCorrelation(ConsensusMap & input_map, double ratio_threshold, String & acc_filter, String & desc_filter) nogil except +
         void normalizeMaps(ConsensusMap & input_map, libcpp_vector[double] & ratios) nogil except +
 

